### ISSUE TITLE
Create a new site for Kivy - ideas and suggestions

### ISSUE THREAD

**Issue by dessant** [kivy/kivy-website#52]
This issue will be mainly used for discussing possible design routes for our new site.

Please comment here if you have any suggetions about how the site should look like or what it should contain, links or screenshots that illustrate your idea are recommended.
##

**Comment by dessant** [comment]
bookmark https://twitter.com/kdamica/status/549977972417892352

**Comment by frastlin** [comment]
I really like the basic template with a short list of links, headings that say why I should use Kivy, that it is MIT and uses OpenGL.
If you wish to show how easy it is, there should be a link where people who are already python developers can read a brief overview of specs, then do the typical hello world example, then read a brief description of how Kivy is structured, how to read the API, and a link to the API/Docs on the front page for super fast access like pygame has (Love the way pygame's API is setup FYI, horrible for newbies, but great for someone who just needs to know what to do to pan a sound to the left).

On the front page though, I don't think we should have more than 10 links in the side bar. I like the simplicity and header and list delineation that is currently in place.
On the bottom of the page there should be a link to YouTube, Google+, Facebook, Twitter and what ever else.
I don't know what the website uses, but just FYI, in WordPress, there are some very nice tools for managing all the social networks at once.

For newbies, There should be a "learn python with Kivy" tutorial. There you can point to other tutorials as further and more advanced reading, but the last thing a potential game developer who stumbles on Kivy needs is a 60 module tutorial telling them how to use python, and nothing on how to use Kivy, then a quite extensive and over-whelming description of Kivy architecture and API. 

Combining the two together (for newbies and python developers) just leads to confusion.

I wouldn't have any videos that are not Kivy made or on Kivy's channel on the front page. The page should be super simple, sleek and, like Kivy, nearly the same on mobile and desktop.

**Comment by kived** [comment]
New website should have both stable and dev docs, instead of just dev. :)

**Comment by dessant** [comment]
@inclement proposed an example gallery like http://matplotlib.org/gallery.html (ignore the theme), it would definitely help newcomers.

**Comment by inclement** [comment]
I only mentioned that because @matham said the same thing earlier in this discussion and I think it's an extremely good idea, especially since we have many examples already. Give him the credit!

**Comment by dessant** [comment]
Sorry, credit given to @matham.

**Comment by hansent** [comment]
less is more for the main web site i think.  at least for me, when i find a new project i'm evaluating, if there is a whole bunch of information on the first page, I often dismiss it as too complicated pretty quickly.

...I realize this is very abstract input to this discussion and doesnt ofer that much practical ideas...but just a thought i guess.

**Comment by frastlin** [comment]
I agree with hansent.
If we wish to have lots of stuff, it should either feel like a different website, or be a different website. In every page, there should not be more than 7 headings. The only exception may be the user guide. I like my user guides in one large HTML page, but even then, if you had 1.1, 1.2, 1.3...1.17.5, 1.17.6... it just gets dumb. I like pyinstaller's documentation, but everything else should just be simple simple simple. Like [Learn python the hard way's page!](http://learnpythonthehardway.org/book/)

**Comment by victor-rene** [comment]
Can we remove the bird? The kiwi is fun, but I had some people really confused about it, especially the spelling of it. Like it's almost the same but not really. What do you think? Can we have a better mascot?

**Comment by FeralBytes** [comment]
I feel @victor-rene really [hit the nail on the head here](https://github.com/kivy/kivy-website/issues/52#issuecomment-68142438).
It would be great to showcase some of Kivy's other projects, if the documentation for them was not super hard to come by. Just read the docs. Perfect example Plyer. Can any one actually tell me how to use plyer.notification or sms; I know that you guys can; and I did figure it out, but not without directly looking at the code in the plyer github. The subprojects need strong documentation too; even if they are new; at least before you can go putting them on a front page.
He is also right about the release gap. Can we get some sort of plan together to figure out when 1.9.0 will come out, or at least make a 1.8.1 that includes some of the many fixes that are critical? IE sound.on_stop never gets called.

**Comment by brousch** [comment]
For plyer, the examples provided with each facade are the best
documentation. They show how to use the facade in a real (small) app.

On Fri, Feb 13, 2015 at 4:22 AM, FeralBytes <notifications@github.com>
wrote:

> I feel @victor-rene <https://github.com/victor-rene> really hit the nail
> on the head here
> <https://github.com/kivy/kivy-website/issues/52#issuecomment-68142438>.
> It would be great to showcase some of Kivy's other projects, if the
> documentation for them was not super hard to come by. Just read the docs.
> Perfect example Plyer. Can any one actually tell me how to use
> plyer.notification or sms; I know that you guys can; and I did figure it
> out, but not without directly looking at the code in the plyer github. The
> subprojects need strong documentation too; even if they are new; at least
> before you can go putting them on a front page.
> He is also right about the release gap. Can we get some sort of plan
> together to figure out when 1.9.0 will come out, or at least make a 1.8.1
> that includes some of the many fixes that are critical? IE sound.on_stop
> never gets called.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kivy/kivy-website/issues/52#issuecomment-74225982>.
>



-- 
 Ben Rousch
   brousch@gmail.com
   http://clusterbleep.net/

**Comment by FeralBytes** [comment]
@brousch but that is the problem only GPS has an example. As another for instance the class is plyer.facades.Notification but that is not how you use it, instead you use it like plyer.notification. Because it is a proxy but you don't know that until you look [here](https://github.com/kivy/plyer/blob/master/plyer/__init__.py). So more examples and documentation would be good for the sub projects.
Again I know documentation is all about time. I am just making the point that before you put the sub projects on the front page, you might want to make them shine first.

**Comment by FeralBytes** [comment]
@brousch Yes it is at the top of the documentation (about the proxy objects that is), but with out the example to fill the gap, I went to the code to figure it out. But most people are not that inclined even if they should be to be programming.

**Comment by brousch** [comment]
I'm not sure what you're looking at, but our GSOC student added examples
for every facade we provide last summer:
https://github.com/kivy/plyer/tree/master/examples

On Fri, Feb 13, 2015 at 6:34 AM, FeralBytes <notifications@github.com>
wrote:

> @brousch <https://github.com/brousch> Yes it is at the top of the
> documentation (about the proxy objects that is), but with out the example
> to fill the gap, I went to the code to figure it out. But most people are
> not that inclined even if they should be to be programming.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kivy/kivy-website/issues/52#issuecomment-74240905>.
>



-- 
 Ben Rousch
   brousch@gmail.com
   http://clusterbleep.net/

**Comment by FeralBytes** [comment]
@brousch https://plyer.readthedocs.org/en/latest/
The documentation.

**Comment by victor-rene** [comment]
A quick update that probably belongs in [kivy-ideas](https://github.com/kivy/kivy-ideas). Let's have a section "Contributions wanted", and list the things that we would like to be done, but can't afford to do because of time/knowledge issues. I'm not talking about bugs, but more about feature gaps that would make Kivy more well rounded.

A recent example is #3048 (don't know if related to https://gist.github.com/tshirtman/6465713). Another one would be TableLayout, etc... "Contributions wanted", if constantly updated will probably help channeling the random programming efforts put into Kivy. My 2 cents.

**Comment by ivlevdenis** [comment]
it is important that the site and documentation was multilingual
